### PR TITLE
Quickmount - Default enable quick mount (empty keybind) and Move to CBA Settings

### LINKED ISSUE #6516: Enabling quick mount should be a one-step process
Simple little suggestion for quick mount: at the moment to enable using quick mount the client has to bind the hotkey _and_ enable it in their ACE settings. Most people I run into have the issue of not knowing they need to enable it. Ideally it should just work off of the hotkey - dunno why I didn't make it like that from the start, but nevertheless it would be sweet.

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Close #6516 

Hotkey is unset by default, no reason to have a setting in that case as described in #6516.

@jameslkingsley review please.

### CODE REVIEW

**Comment by Dystopian** [comment]
What if some server admin would like to disable this feature? I think jameslkingsley meant just enabling by default.

**Comment by jonpas** [comment]
But he also said that's a hacky fix. :D

**Comment by jameslkingsley** [comment]
Looks good to me 👍

**Comment by Dystopian** [comment]
I just think Quick Mount can be not appropriate for some milsim PvP games because it's cheating a little. Setting would be helpful in these cases.
BTW where is that jonpas who asked me to add setting for some features I implemented? ;)

**Comment by jonpas** [comment]
I don't know, not my feature, I'll do what others think is best.

**Comment by jonpas** [comment]
`movedToSQF = 1;`

**Comment by PabstMirror** [comment]
ace_common preinit runs first
if someone has a modified ace_settings in either mission config or configFile, the settings converted would think it was just an unconverted setting and try to add it first

File: addons/quickmount/initSettings.sqf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[
+    QGVAR(enabled),
+    "CHECKBOX",
+    localize ELSTRING(common,Enabled),
+    format ["ACE %1", localize LSTRING(Category)],
```
Comment (Dystopian) [inline addons/quickmount/initSettings.sqf]: `LLSTRING` and `LELSTRING` could be used here and anywhere

Comment (commy2) [reply addons/quickmount/initSettings.sqf]: Could simply delete the `localize` , because the settings system can translate the strings on its own.

Comment (Dystopian) [reply addons/quickmount/initSettings.sqf]: btw just checked, subcategory is not automatically localized

Comment (commy2) [reply addons/quickmount/initSettings.sqf]: Issue report then.

Comment (PabstMirror) [reply addons/quickmount/initSettings.sqf]: fixed in https://github.com/CBATeam/CBA_A3/pull/1003

**Pull request merged**